docs/get-started-angular.md

```diff
@@ -0,0 +1,351 @@
+            ],
+            // ...
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+Note that although standalone surveys support [multiple predefined themes](https://surveyjs.io/Documentation/Library?id=get-started-angular#configure-styles), surveys within Survey Creator apply only one theme ("Default V2"). However, you can use the survey configuration (JSON object) produced by Survey Creator to build a standalone survey and apply any theme to it. You can also customize Survey Creator theme colors as shown in the following example:
+
+[View Customize Theme Using CSS Variables example](https://surveyjs.io/Examples/Survey-Creator?id=editor-custom-theme-css-vars)
+## Configure Survey Creator
+
+To configure the Survey Creator component, specify [its properties](https://surveyjs.io/Documentation/Survey-Creator?id=surveycreator) in a configuration object. In this tutorial, the object enables the following properties:
+
+- [`showLogicTab`](https://surveyjs.io/Documentation/Survey-Creator?id=surveycreator#showLogicTab)        
+Displays the Logic tab in the tab panel.
+
+- [`isAutoSave`](https://surveyjs.io/Documentation/Survey-Creator?id=surveycreator#isAutoSave)        
+Automatically saves the survey definition JSON on every change.
+
+```js
+const creatorOptions = {
+    showLogicTab: true,
+    isAutoSave: true
+};
+```
+
+Pass the configuration object to the `SurveyCreator` constructor as shown in the code below to instantiate Survey Creator. Assign the produced instance to a constant that will be used later to render the component:
+
+```js
+import { Component, OnInit } from '@angular/core';
+import { SurveyCreator } from "survey-creator-knockout";
+
+@Component({
+  selector: 'survey-creator',
+  templateUrl: './survey-creator.component.html',
+  styleUrls: ['./survey-creator.component.css']
+})
+export class SurveyCreatorComponent implements OnInit {
+  ngOnInit() {
+    const creator = new SurveyCreator(creatorOptions);
+  }
+}
+```
+
+<details>
+  <summary>View full code</summary>
+
+```js
+import { Component, OnInit } from "@angular/core";
+import { SurveyCreator } from "survey-creator-knockout";
+
+const creatorOptions = {
+  showLogicTab: true,
+  isAutoSave: true
+};
+
+@Component({
+  selector: 'survey-creator',
+  templateUrl: './survey-creator.component.html',
+  styleUrls: ['./survey-creator.component.css']
+})
+export class SurveyCreatorComponent implements OnInit {
+  ngOnInit() {
+    const creator = new SurveyCreator(creatorOptions);
+  }
+}
+```
+</details>
+
+## Render Survey Creator
+
+Switch to the component template. Add a page element that will serve as the Survey Creator container:
+
+```html
+<div id="surveyCreator"></div>
+```
+
+```css
+#surveyCreator {
+    height: 100vh;
+    width: 100vw;
+}
+```
+
+To render Survey Creator in the page element, call the `render(containerId)` method on the Survey Creator instance you created in the previous step:
```

Comment:
Is "on the Survey Creator instance" correct, or do you mean "in the Survey Creator instance"?

Reply:
https://www.quora.com/What-does-it-mean-to-call-a-method-upon-an-object/answer/Steven-2225

I know it's just a user answer, but its the best explanation I found.

Reply:
Thanks for the clarification, Roman.